plugins/outputs/awscsm/awscsm.go

```diff
@@ -146,7 +146,10 @@ func (c *CSM) Connect() error {
 	c.publishingOffset = time.Duration(rand.Int63n(int64(60 * time.Second)))
 
 	commonCreds := credentialConfig.Credentials()
-	commonCfg := commonCreds.ClientConfig(csm.ServiceName, &aws.Config{})
+	commonCfg := commonCreds.ClientConfig(csm.ServiceName, &aws.Config{
+		LogLevel: configaws.SDKLogLevel(),
+		Logger:   configaws.SDKLogger{},
```

Comment:
Basically anywhere you search and find an `aws.Config{}` object being instantiated you will now see we set the `LogLevel` and `Logger`.